api/v1alpha08/sonataflowplatform_types.go

```diff
@@ -45,7 +45,23 @@ type SonataFlowPlatformSpec struct {
 	// Only workflows without the `sonataflow.org/profile: dev` annotation will be configured to use these service(s).
 	// Setting this will override the use of any cluster-scoped services that might be defined via `SonataFlowClusterPlatform`.
 	// +optional
-	Services *ServicesPlatformSpec `json:"services,omitempty"`
```

Comment:
this should not be changed. please revert this back to a pointer object. it is critical that this be a pointer object in order for the cluster-wide services logic to work as expected.

why are we making changes to this? it has nothing to do w/ the PR.

Reply:
Not sure how this came here, definitely something I did by mistake but not a problem to revert it.

Reply:
Done.